src/mozanalysis/metrics/desktop.py

```diff
@@ -7,6 +7,24 @@
 # in https://github.com/mozilla/metric-hub
 
 from mozanalysis.config import ConfigLoader
+import warnings
+
+warnings.simplefilter("default")
+warnings.warn(
+    """
+    metrics and data sources created in mozanalysis are deprecated
```

Comment:
If we're adding the same warning to a bunch of different files, should we abstract that string out to a single place and just import into each file? something like:

```python
from mozanalysis.warning_messages import mozanalysis_metric_deprecation

warnings.simplefilter("default")
warnings.warn(mozanalysis_metric_deprecation)
```

in each file and then the actual warning string defined in `warning_messages` or something

Reply:
Good idea, instead of importing the warning message I just put it in the `__init__.py` so that if you import any of the metric files it raises the warning.